plugins/iterm2/iterm2.plugin.zsh

```diff
@@ -0,0 +1,38 @@
+#####################################################
+# iTerm2 plugin for oh-my-zsh                       #
+# Author: Aviv Rosenberg (github.com/avivrosenberg) #
+#####################################################
+
+###
+# This plugin is only relevant if the terminal is iTerm2 on OSX.
```

Comment:
Could be relevant on other OSes too, if you are SSHing in to them from iTerm2 on OS X. Don't know how to detect that, though, since `ssh` normally doesn't propagate `$TERM_PROGRAM`.